hikari/guilds.py

```diff
@@ -551,6 +551,21 @@ class Role(PartialRole):
     and increase as you go up the hierarchy.
     """
 
+    bot_id: typing.Optional[snowflakes.Snowflake] = attr.ib(eq=False, hash=False, repr=True)
+    """The ID of the bot this role belongs to.
+
+    If `builtins.None`, this is not a bot role.
+    """
+
+    integration_id: typing.Optional[snowflakes.Snowflake] = attr.ib(eq=False, hash=False, repr=True)
+    """The ID of the integration this role belongs to.
+
+    If `builtins.None`, this is not a integration role.
+    """
+
+    is_premium_subscriber_role: bool = attr.ib(eq=False, hash=False, repr=True)
```

Comment:
Might be worth mentioning that this is "boosts" somewhere here as that's what it's more likely going to be known as

Reply:
Also thought about that one, but then scrapped it cause discord specially specified it as "premium subscriber" so they might make it do something else with it in the future. 

Will add it to the docstring tho